spec/integrations/fixtures/rspec/let_it_be_fixture.rb

```diff
@@ -31,7 +31,7 @@
   config.alias_to :let_with_refind, refind: true
 end
```

Comment:
Out of curiosity, do we run those ` spec/integrations/` on CI?
`fd fix spec/integrations/fixtures/rspec | xargs -L1 rspec` failed for me on several examples:
```
  4) Something nested context is expected to eq true
     Failure/Error: specify { is_expected.to eq true }

       expected: true
            got: nil

       (compared using ==)
rspec ./spec/integrations/fixtures/rspec/rspec_stamp_fixture_tmpl.rb:12 # Something fail me
rspec ./spec/integrations/fixtures/rspec/rspec_stamp_fixture_tmpl.rb:24 # Something
    you
    can
    not
    patch me

rspec ./spec/integrations/fixtures/rspec/rspec_stamp_fixture_tmpl.rb:16 # Something fail me with tag
rspec ./spec/integrations/fixtures/rspec/rspec_stamp_fixture_tmpl.rb:35 # Something nested context is expected to eq true
```

```
  1) before_all + Isolator fails when within transaction
     Failure/Error: FailingJob.perform_async(false)

     Isolator::BackgroundJobError:
       You are trying to enqueue background job inside db transaction. In case of transaction failure, this may lead to data inconsistency and unexpected bugs
       Details: FailingJob ([false])
     # /Users/pirj/.rvm/gems/ruby-2.5.1@test-prof/gems/sidekiq-6.0.5/lib/sidekiq/client.rb:74:in `push'
     # /Users/pirj/.rvm/gems/ruby-2.5.1@test-prof/gems/sidekiq-6.0.5/lib/sidekiq/worker.rb:243:in `client_push'
     # /Users/pirj/.rvm/gems/ruby-2.5.1@test-prof/gems/sidekiq-6.0.5/lib/sidekiq/worker.rb:200:in `perform_async'
     # ./spec/integrations/fixtures/rspec/before_all_isolator_fixture.rb:65:in `block (3 levels) in <top (required)>'
     # /Users/pirj/.rvm/gems/ruby-2.5.1@test-prof/gems/activerecord-6.0.2.1/lib/active_record/connection_adapters/abstract/database_statements.rb:281:in `block in transaction'

Finished in 0.04757 seconds (files took 1.65 seconds to load)
3 examples, 1 failure

Failed examples:

rspec ./spec/integrations/fixtures/rspec/before_all_isolator_fixture.rb:62 # before_all + Isolator fails when within transaction
```

```
  1) User fabrication creates and reloads user
     Failure/Error: expect(User.find(user.id).name).to eq "John 1"

       expected: "John 1"
            got: "John 0"

       (compared using ==)
     # ./spec/integrations/fixtures/rspec/factory_doctor_fixture.rb:39:in `block (3 levels) in <top (required)>'

Finished in 0.11925 seconds (files took 1.52 seconds to load)
7 examples, 1 failure

Failed examples:

rspec ./spec/integrations/fixtures/rspec/factory_doctor_fixture.rb:37 # User fabrication creates and reloads user
```

(and segfautled a couple of times, too).
Ruby 2.5.1
isolator (0.6.1) in Gemfile.lock

Reply:
I've figured that out, there are deliberate failures in spec/integration/fixtures, and specs in spec/integration take care of checking the output.